### DISCUSSION TITLE
How do I delete templates?

### DISCUSSION THREAD

**Discussion by axzr** [codesandbox/codesandbox-client#6015] [General]
I seem to have created some templates, though it wasn't intended. How do I delete them?

Cheers.

**Comment by JamesACS** [comment]
Hey @axzr , 

Unfortunately you can't delete templates (these are just templates you've selected to start a project) but once you've started the project, if you then don't like it or otherwise want to dispose of it you can just click and drag it to 'Recently Deleted' on the side bar.

**Comment by axzr** [comment]
Thanks James.

**Comment by joshsutterfield** [comment]
You can find Templates under Projects. In the dropdown for each template there's a link "Close _Project_". "Close" seems to be like "Archive", and "Project" means template. 🤷
